roles/openshift_logging/defaults/main.yml

```diff
@@ -69,15 +69,15 @@ openshift_logging_kibana_ops_key: ""
 openshift_logging_kibana_ops_ca: ""
 
 openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nodeselector_label | default('logging-infra-fluentd=true') | map_from_pairs }}"
-openshift_logging_fluentd_cpu_limit: "{{ openshift_hosted_logging_fluentd_cpu_limit | default('100m') }}"
-openshift_logging_fluentd_memory_limit: "{{ openshift_hosted_logging_fluentd_memory_limit | default('512Mi') }}"
+openshift_logging_fluentd_cpu_limit: 100m
+openshift_logging_fluentd_memory_limit: 512Mi
```

Comment:
I don't think this change is necessary.  It is possible that someone had set `openshift_hosted_logging_fluentd_cpu_limit` in their old inventory or vars file, and they will expect this value to be used when deploying the new version.  Same with the other `openshift_hosted_logging_*` parameters.

I know it is confusing.  1.4 and earlier used `openshift_hosted_logging`.  1.5 uses `openshift_logging`.  1.6 will mostly use `openshift_logging` but some of the roles may have their own specific parameters e.g. `openshift_logging_kibana`, `openshift_logging_fluentd`, etc.